prototype2/tools/udpgen_pcap.cpp

```diff
@@ -1,31 +1,54 @@
 /** Copyright (C) 2016, 2017 European Spallation Source ERIC */
 
+#include <CLI/CLI.hpp>
 #include <arpa/inet.h>
 #include <cassert>
 #include <cinttypes>
-#include <tools/PcapArgs.h>
 #include <tools/ReaderPcap.h>
 #include <common/Socket.h>
 #include <string.h>
 #include <string>
 #include <unistd.h>
 // GCOVR_EXCL_START
+
+struct {
+  std::string FileName{""};
+  std::string IpAddress{"127.0.0.1"};
+  uint16_t UDPPort{9000};
+  uint64_t NumberOfPackets{0}; // 0 == all packets
+  uint64_t SpeedThrottle{0}; // 0 is fastest higher is slower
+  uint64_t PktThrottle{0}; // 0 is fastest
```

Comment:
I think this comment can be slightly misleading as 0 means this throttle is off; 1 would make it slower than 10 then.